packages/react/src/Link/Link.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { ChevronRight } from '@amsterdam/design-system-react-icons'
+import clsx from 'clsx'
+import { AnchorHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react'
+import { Icon } from '../Icon/Icon'
+
+export type LinkOnBackground = 'default' | 'light' | 'dark'
+export type LinkVariant = 'standalone' | 'standaloneBold' | 'inline' | 'inList'
+
+export interface LinkProps extends Omit<AnchorHTMLAttributes<HTMLAnchorElement>, 'placeholder'> {
+  variant?: LinkVariant
+  onBackground?: LinkOnBackground
+}
+
+export const Link = forwardRef(
+  (
+    { children, href, variant, onBackground, className, ...restProps }: PropsWithChildren<LinkProps>,
+    ref: ForwardedRef<HTMLAnchorElement>,
+  ) => (
+    <a
+      href={href}
```

Comment:
I don't think we need this, right? This is already part of `restProps`

Reply:
Need for forwardRef